lib/cli/commands/validate.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const fs = require('fs-extra')
+const path = require('path')
+const pathContains = require('contains-path')
+const Ajv = require('ajv')
+
+const supportSchema = require('./../../../schema.json')
+
+module.exports = function (opts) {
+  return {
+    command: 'validate',
+    desc: 'Validate support information schema',
+    builder: function (yargs) {
+      return yargs
+    },
+    handler: async function (argv) {
+      let userPackageJson
+      try {
+        userPackageJson = await fs.readJSON(path.join(process.cwd(), 'package.json'))
```

Comment:
this won't work if it's invoked in a nested dir, but also, find for now

Reply:
Do you think [pkg-up](npmjs.com/package/pkg-up) would be better for this task?